extensions/oauth/src/test/resources/authorization-code/authorization-code-localcallbackref-config.xml

```diff
@@ -14,22 +14,21 @@
 
     <oauth:token-manager-config name="tokenManagerConfig"/>
 
+    <oauth:authorization-code-grant-type
```

Comment:
The problem with that approach is that Mozart won't be able to use it since they don't handle top level elements.

That been said, the OAuth module does require the manager and other global stuff to be registered and it's not among the most common use cases. So maybe this is fine and Mozart will have to eventually overcome this limitation. 

We can also review the lifecycle again or find a third alternative. Awaiting for @pablolagreca 's input